test/cases/azure.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
+set -euo pipefail
+
+source /etc/os-release
+
+# Colorful output.
+function greenprint {
+    echo -e "\033[1;32m${1}\033[0m"
+}
+
+#TODO: Remove this once there is rhel9 support for Azure image type
+if [[ $DISTRO_CODE == rhel_90 ]]; then
+    greenprint "Skipped"
+    exit 0
+fi
+
+# Provision the software under tet.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+# We need azure-cli to talk to Azure.
```

Comment:
Do we? I don't see it being used anywhere.

Reply:
Terraform needs it. `Error building AzureRM Client: Please ensure you have installed Azure CLI version 2.0.79 or newer. Error parsing json result from the Azure CLI: Error launching Azure CLI: exec: "az": executable file not found in $PATH.`

Reply:
Does it need to be installed explicitly or only the repos configured so that terraform can pull it as a dependency? In any case a comment saying TF needs this is a good idea.

Reply:
It needs to be installed, it won't look for it unless it tries to process the `main.tf` file and then it just gives error if it can't find it. I updated the comment.